traffic_ops/app/lib/Extensions/TrafficStats/API/CdnStats.pm

```diff
@@ -52,7 +52,7 @@ sub get_usage_overview {
 
 	my ( $rc, $result ) = $cstats->get_usage_overview();
 	if ( $rc == SUCCESS ) {
-		return $self->success($result);
+		return $self->deprecation_with_no_alternative(200, $result);
 	}
 	else {
 		return $self->alert($result);
```

Comment:
This won't return the deprecation notice in the event of an error.

Reply:
good point